mongodump/mongodump_test.go

```diff
@@ -734,6 +694,70 @@ func TestMongoDumpBSON(t *testing.T) {
 	})
 }
 
+func TestMongoDumpBSONLongCollectionName(t *testing.T) {
```

Comment:
I had to toss this under a new test since I don't believe we can do `t.Skip()` for just one sub-call of `Convey()`.